internal/database/offline/sqlutil/export.go

```diff
@@ -26,6 +26,49 @@ func Export(ctx context.Context, db *sqlx.DB, opt offline.ExportOpt, backend typ
 		Backend: backend,
 		SqlxDB:  db,
 	}
+	for groupID, snapshotTable := range opt.SnapshotTables {
+		if snapshotTable == types.TempTable {
+			tableName := dbutil.TempTable(fmt.Sprintf("snapshot_%d", groupID))
+			features := opt.Features[groupID]
+			entity := features[0].Entity()
+
+			schema := ""
+			// export will failed since mysql will trigger an error when you refer to a temporary table more than one time in a single query
+			// see: https://dev.mysql.com/doc/refman/8.0/en/temporary-table-problems.html
+			if backend == types.BackendMySQL {
```

Comment:
should we drop mysql "temporary" tables?

Reply:
I should drop the `temporary` table, but I just didn't think of how to do it. What if I drop it in `sqlxQueryExportResults`?

Reply:
it just occurred to me that I might not need to introduce a temporary table here. I'll just create new normal tables and drop them one by one in `sqlxQueryExportResults`. What do you think?